components/pipeline-service/tekton-results/kustomization.yaml

```diff
@@ -2,17 +2,20 @@ resources:
 - allow-argocd-to-manage.yaml
 - tekton-pipelines-namespace.yaml
 - db-execution-configmap.yaml
-- https://github.com/tektoncd/results/releases/download/v0.4.0/release.yaml
+- https://github.com/openshift-pipelines/tektoncd-results.git/config?ref=downstream
```

Comment:
pin to git-sha so that we can fully manage the version - so it should not happen that downstream branch is updated without testing on our side.

Reply:
Updated. Thanks.